dll/win32/msi/handle.c

```diff
@@ -246,8 +246,8 @@ int msiobj_release( MSIOBJECTHDR *info )
     {
         if( info->destructor )
             info->destructor( info );
-        msi_free( info );
         TRACE("object %p destroyed\n", info);
```

Comment:
It's okay to print a pointer with %p even if that pointer is invalid. The print will just show the address it points to, not dereference it.